app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is a class variable that holds references to all instances of `Animal`. Ensure that this is the intended behavior, as it could lead to memory leaks if not managed properly.